### ISSUE TITLE
Running interactive command in shell

### ISSUE THREAD

**Issue by mdhorn** [VladimirMarkelov/clui#116]
I have a situation where I'm stuck running a different interactive command in a shell from my clui-go program.  When exit the spawned program, the clui display is not properly redrawing.

What calls should I be doing to save the current state of the layout before running the shell program? And recommendation on how to properly restore the state of the layout after?

I was hoping to find a simple "drop to bash shell" example, but no luck.

**Comment by VladimirMarkelov** [comment]
It is possible that underlining termbox does not redraw anything. It tries to minimize repaintnig, and keeps areas that have been changed between redraw events(if you put the same symbol to the same cell termbox does not treat this as change - it looks for difference). But it does not know anything about external applications. Maybe it needs some call to reset internal termbox cache - though, there is no similar call in clui.

I do not have a simple example at hand to play with. I found a function `Sync` in termbox (in termbox-go/api.go) that may help. Try to `import "github.com/nsf/termbox-go"` and call its `Sync` after the spawned program finishes. If it works we can add a wrapper to `clui` that will call termbox internally.
